_data/chains/eip155-314.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Filecoin — Mainnet",
+  "chain": "FIL",
+  "status": "incubating",
+  "rpc": [],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FIL",
+    "symbol": "FIL",
+    "decimals": 18
+  },
+  "infoURL": "https://filecoin.io",
+  "shortName": "filecoin",
+  "icon": "filecoin",
+  "chainId": 314,
+  "networkId": 314,
+  "explorers": [
```

Comment:
We need to add:
`"slip44": 461,`
(Filecoin was added by Zondax/Glif/MetaMask in 2019, see https://github.com/satoshilabs/slips/blob/master/slip-0044.md)

Reply:
Done here: [`18f8242` (#1567)](https://github.com/ethereum-lists/chains/pull/1567/commits/18f8242a0e31649f441e7694d470206bc7bf50a6)